content/200-concepts/100-components/01-prisma-schema/06-relations/420-referential-integrity.mdx

```diff
@@ -8,21 +8,53 @@ preview: true
 
 <TopBlock>
 
-Referential integrity means a set of constraints or triggers to handle the consistency of data of a [`@relation`](/reference/api-reference/prisma-schema-reference#relation). It can mean preventing a deletion of a record referred from other records, or it can be a cascading action defined in the field's [`referential actions`](/concepts/components/prisma-schema/relations/referential-actions). 
+To keep data consistent, it is necessary to have rules for what happens when related records are deleted or updated. For example, a `Post` model will need to define what happens if its author is deleted. **Referential integrity** refers to the set of constraints and triggers used to enforce this consistency.
```

Comment:
While the last sentence is kinda technically correct, I am a bit weirded out that this looks like a definition. And I would define "ref integrity" more like Wikipedia does it: https://en.wikipedia.org/wiki/Referential_integrity (ALso meaning: Contraints and triggers are tools that can help enforce referential integrity - but generally it is an abstract concept instead of a function or tool)

Reply:
I've made a better attempt at this intro using your comment and @pimeys above... happy for more suggestions...